release/ray_release/byod/build.py

```diff
@@ -61,6 +63,25 @@ def build_anyscale_byod_images(tests: List[Test]) -> None:
                     time.sleep(30)
                     timeout += 30
                     continue
+                subprocess.check_call(
+                    [
+                        "docker",
+                        "build",
+                        "--build-arg",
+                        f"BASE_IMAGE={byod_image}",
+                        "--build-arg",
+                        "PIP_REQUIREMENTS=requirements_byod.txt",
+                        "--build-arg",
+                        "DEBIAN_REQUIREMENTS=requirements_debian_byod.txt",
+                        "-t",
+                        byod_image,
+                        "-f",
+                        "byod.Dockerfile",
+                        os.join.path(RELEASE_PACKAGE_DIR, "ray_release/byod"),
+                    ],
+                    stdout=subprocess.DEVNULL,
+                    env={"DOCKER_BUILDKIT": "1"},
```

Comment:
you probably would want to inherit the `os.environ`, by making a copy and then add `DOCKER_BUILDKITE=1`. otherwise, this will drop the other env variables.

or, maybe just add `DOCKER_BUILDKITE=1` into the pipeline environment.